client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminCheckout.tsx

```diff
@@ -0,0 +1,123 @@
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+import YourOrder from '../../cart/YourOrder';
+import {
+  selectCartContents,
+  selectDiscount,
+} from '../../ticketingmanager/ticketing/ticketingSlice';
+import {useAppSelector} from '../../app/hooks';
+import {loadStripe} from '@stripe/stripe-js';
+import {ReactElement, useState} from 'react';
+import AdminCompleteOrderForm, {
+  CheckoutFormInfo,
+} from './AdminCompleteOrderForm';
+import {selectDonation} from '../../ticketingmanager/donationSlice';
+import {useNavigate} from 'react-router-dom';
+
+const pk = `${process.env.REACT_APP_PUBLIC_STRIPE_KEY}`;
+const stripePromise = loadStripe(pk);
+
+/**
+ * Displays Checkout Page
+ *
+ * @returns {ReactElement}
+ */
+export default function CheckoutPage(): ReactElement {
+  const navigate = useNavigate();
+  const cartItems = useAppSelector(selectCartContents);
+  const discount = useAppSelector(selectDiscount);
+  const donation = useAppSelector(selectDonation);
+  const handleBackButton = () => {
+    navigate('/ticketing/purchaseticket');
+  };
+  const doCheckout = async (formData: CheckoutFormInfo) => {
+    if (formData.seatingAcc === 'Other') {
+      formData.seatingAcc = formData.comments;
+    }
+    const stripe = await stripePromise;
+    if (!stripe) return;
+    const response = await fetch(
+      process.env.REACT_APP_API_1_URL + `/events/checkout`,
+      {
+        credentials: 'include',
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({cartItems, formData, donation, discount}),
+      },
+    );
+    const session = await response.json();
+    console.log(session.id);
+    console.log(pk);
+    const paymentIntent = session.payment_intent;
+    const result = await stripe.redirectToCheckout({sessionId: session.id});
+    if (result.error) {
+      console.log(result.error.message);
+    }
+  };
+
+  return (
+    <div className='w-full h-screen overflow-x-hidden absolute'>
+      <div className='mt-[5rem] md:ml-[15rem] md:mr-[2rem] md:mt-20 sm:mt-[5rem] sm:ml-[1rem] sm:mr-[1rem] sm:mb-[11rem]'>
+        <div className='flex flex-row'>
+          <div
+            className='bg-zinc-200 flex flex-col md:flex-col sm:flex-col
+         max-md:items-center w-full h-full p-2 lg:p-[5rem] md:p-[1rem]'
+          >
```

Comment:
You can actually combine all three of these `div`s into a single div with these classes: `flex flex-col md:ml-[18rem] tab:mx-[5rem] mx-[1.5rem] mt-[5rem] mb-[9rem]`. This would also get rid of the gray box and extra spacing around the outside

Reply:
Changed then corrected margin for "back to Purchase" button
![image](https://github.com/WonderTix/WonderTix/assets/97830813/3f5fd24d-8eed-468b-b18f-df6203c2bb06)
![image](https://github.com/WonderTix/WonderTix/assets/97830813/0294ae12-ea1f-4463-bdb0-e58507d51110)